src/profiler.h

```diff
@@ -191,6 +192,8 @@ class Profiler {
     void shutdown(Arguments& args);
     Error check(Arguments& args);
     Error start(Arguments& args, bool reset);
+    Error setEcid(const char* ecid);
+    const char* getEcid();
```

Comment:
`ECID` does not seem to be a commonly used abbreviation.

Reply:
I believe it is used in many Oracle technologies, but I can switch the name to simply correlationId.

Reply:
Something like "ContextID" might be clear?

Reply:
done